README.md

```diff
@@ -1,13 +1,31 @@
-<img height="256" align="left" src="https://user-images.githubusercontent.com/39812919/122685073-b5075180-d1df-11eb-8172-9ee6ada53190.gif" />
+## 👩 Hi!
 
-#### about me
+Eu sou Isabella Basso
+ 
+Redpill 💊 + Desenvolvedora
 
-- learning C + django
-- software engineer @ Red Hat Plumbers team
-- undergrad student @ [universidade de são paulo](https://www5.usp.br)
-- proud member of [lkcamp.dev](LKCamp)
+Cursando Ciências Moleculares 🧪 - USP.
 
-#### what i do
+Meu foco atualmente está em conseguir um blahaj 🦈 e em segundo plano em fazer bugs baixo nível com C (odeio Rust btw 🦀🔫)
 
-- now: C static analysis tool + linux kernel unit testing
-- previous: whatsapp bot + AR for the web
+<div>
+  <img height="180em" src="https://github-readme-stats.vercel.app/api?username=isinyaaa&show_icons=true&theme=tokyonight"/>
+  <img height="180em" src="https://github-readme-stats.vercel.app/api/top-langs/?username=isinyaaa&layout=compact&theme=tokyonight"/>
+</div>
+
+### ❤️ Let's get connected:
```

Comment:
Fixup this to match my above comment. I.e. should be
```suggestion
## ❤️ Let's get connected:
```

Reply:
Are you sure this should be done in English? I think it'd be best to use a single language throughout...